src/rules/resolutions.ts

```diff
@@ -15,8 +15,14 @@ import * as core from '@actions/core';
 import { PackageJson } from 'type-fest';
 
 export const validateResolutions = (packageJson: PackageJson) => {
-  const ignoredResolutions = core.getMultilineInput('ignore-resolutions');
+  const ignoreUntilDate = new Date(core.getInput('ignore-resolutions-until'));
```

Comment:
What happens when `core.getInput('ignore-resolutions-until')` returns undefined? I think we should explicitly handle the case when this input is not provided

Reply:
That's actually a good shout. Have updated the logic and added a relevant test [here](https://github.com/ExpediaGroup/package-json-validator/pull/291/commits/2dd0c089fb20399690b261ca4fdbd23bd741fda4).

Not a huge fan of nested `if` statements, but feels fairly all right for this use case.